media-sound/cantata/cantata-2.0.1.recipe

```diff
@@ -0,0 +1,110 @@
+SUMMARY="A Qt based client for Music Player Daemon"
+DESCRIPTION="
+Music Player Daemon (MPD) is a flexible, powerful, server-side application \
+for playing music. Through plugins and libraries it can play a variety \
+of sound files while being controlled by its network protocol.
+
+Cantata is a full featured Qt based client for Music Player Daemon."
+HOMEPAGE="https://github.com/CDrummond/cantata"
+COPYRIGHT="2010-2016 Craig Drummond"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/CDrummond/cantata/archive/v2.0.1.tar.gz"
+CHECKSUM_SHA256="294fc96944dd2bb6056525a49ae9db1b1da1b7902ac63286ed6c01622b8e3b52"
+PATCHES="cantata-$portVersion.patchset"
+ADDITIONAL_FILES="cantata.rdef"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	cantata$secondaryArchSuffix = $portVersion
+	app:cantata$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	ffmpeg$secondaryArchSuffix
+	qt5${secondaryArchSuffix}
+	lib:libmpdclient$secondaryArchSuffix
+	lib:libmpg123$secondaryArchSuffix
+	lib:libtag$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	ffmpeg${secondaryArchSuffix}_devel
+	qt5${secondaryArchSuffix}_devel
+	devel:libmpdclient$secondaryArchSuffix
+	devel:libmpg123$secondaryArchSuffix
+	devel:libtag$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc${secondaryArchSuffix}
+	cmd:ld${secondaryArchSuffix}
+	cmd:make
+	cmd:pkg_config${secondaryArchSuffix}
+	"
+
+#Qt cmake folders global definition
+export Qt5Concurrent_DIR="/boot/system/data/cmake/Modules/Qt5Concurrent"
+export Qt5Core_DIR="/boot/system/data/cmake/Modules/Qt5Core"
+export Qt5DBus_DIR="/boot/system/data/cmake/Modules/Qt5DBus"
+export Qt5LinguistTools_DIR="/boot/system/data/cmake/Modules/Qt5LinguistTools"
+export Qt5Multimedia_DIR="/boot/system/data/cmake/Modules/Qt5Multimedia"
+export Qt5Network_DIR="/boot/system/data/cmake/Modules/Qt5Network"
+export Qt5Test_DIR="/boot/system/data/cmake/Modules/Qt5Test"
+export Qt5Sql_DIR="/boot/system/data/cmake/Modules/Qt5Sql"
+export Qt5Svg_DIR="/boot/system/data/cmake/Modules/Qt5Svg"
+export Qt5Xml_DIR="/boot/system/data/cmake/Modules/Qt5Xml"
+export Qt5Widgets_DIR="/boot/system/data/cmake/Modules/Qt5Widgets"
```

Comment:
Korli, i'm using gcc2h for my ports, x64 Haiku installed just for testing. I just built the latest cmake from haikuports, but it stil lrequires theese exports. Should i try to rebuild Qt too for gcc2h?